provider/aws/service.go

```diff
@@ -149,11 +149,36 @@ func (p *AWSProvider) ServiceGet(name string) (*structs.Service, error) {
 		}
 	}
 
+	switch s.Tags["Service"] {
+	case "mysql":
+		s.Exports["URL"] = fmt.Sprintf("mysql://%s:%s@%s:%s/%s", s.Outputs["EnvMysqlUsername"], s.Outputs["EnvMysqlPassword"], s.Outputs["Port3306TcpAddr"], s.Outputs["Port3306TcpPort"], s.Outputs["EnvMysqlDatabase"])
+	case "papertrail":
+		s.Exports["URL"] = s.Parameters["Url"]
+	case "postgres":
+		s.Exports["URL"] = fmt.Sprintf("postgres://%s:%s@%s:%s/%s", s.Outputs["EnvPostgresUsername"], s.Outputs["EnvPostgresPassword"], s.Outputs["Port5432TcpAddr"], s.Outputs["Port5432TcpPort"], s.Outputs["EnvPostgresDatabase"])
+	case "redis":
+		s.Exports["URL"] = fmt.Sprintf("redis://%s:%s/%s", s.Outputs["Port6379TcpAddr"], s.Outputs["Port6379TcpPort"], s.Outputs["EnvRedisDatabase"])
+	case "s3":
+		s.Exports["URL"] = fmt.Sprintf("s3://%s:%s@%s", s.Outputs["AccessKey"], s.Outputs["SecretAccessKey"], s.Outputs["Bucket"])
+	case "sns":
+		s.Exports["URL"] = fmt.Sprintf("sns://%s:%s@%s", s.Outputs["AccessKey"], s.Outputs["SecretAccessKey"], s.Outputs["Topic"])
+	case "sqs":
+		if u, err := url.Parse(s.Outputs["Queue"]); err == nil {
+			u.Scheme = "sqs"
+			u.User = url.UserPassword(s.Outputs["AccessKey"], s.Outputs["SecretAccessKey"])
+			s.Exports["URL"] = u.String()
+		}
+	case "webhook":
+		if parsedUrl, err := url.Parse(s.Parameters["Url"]); err == nil {
+			s.Exports["URL"] = parsedUrl.Query().Get("endpoint")
+		}
+	}
+
 	// Populate linked apps
 	for k, _ := range s.Outputs {
 		if strings.HasSuffix(k, "Link") {
 			n := DashName(k)
-			app := n[:len(n)-5]
+			app := n[:len(n)-4]
```

Comment:
This actually seems to leave a trailing dash now. What was the problem with `-5`?